patternfly/patternfly-elements
# feat(modal): PF1:1

## What I did
- deprecate the `width` attribute in favour of `variant`
- implement many `--pf-` css variables
- remove _all_ `--pfe-` css variables

## Testing Instructions
- Load up the deploy preview for this branch
- Load up [PatternFly v4 Modal](https://www.patternfly.org/v4/components/modal/)
- Compare and contrast
- Read over the [docs](https://deploy-preview-2036--patternfly-elements.netlify.app/components/modal/) and see if anything's missing or no longer relevant

## Notes to reviewers
This PR is *BREAKING*. It aligns pfe-modal with pfv4. the previously-pfe-specific styles and APIs will be moved over to https://github.com/RedHat-UX/red-hat-design-system/pull/288
So dip into the percy tests to see if anything wild broke
